deploy/helm/rawfile-localpv/values.yaml

```
@@ -6,6 +6,11 @@ logLevel: INFO
 # -- Format of the logs (json, pretty)
 logFormat: json
 
+# -- Reserved storage amount on each node (Supports percentage and size) [e.g. `25%` or `50GB` or `10MiB`]
+reservedStorage: 0B
+# -- Overrides total capacity of the storage (Support size values) [e.g. `50GB` or `10MiB`]
+capacityOverride: 0B
```

I don't like having these special cases where 0 means something different than 0 :)

```suggestion
capacityOverride: ""
```

---

nit: on the naming, how about: `allowedStorage` or `allowedCapacity` ?